Master-Game/World/BaseCollectible.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace MasterGame.World
+{
+    public class BaseCollectible
+    {
+        public TileType Type { get; }
+        public bool Passable { get; }
+        public bool Occupied { get; set; }
+        public int Elevation { get; }
+        public float Damage { get; }
+        public int X { get; set; }
+        public int Y { get; set; }
+
+        public BaseCollectible(CollectibleType type, bool passable, float damage)
```

Comment:
I would have the X and Y attribute be passed in when called. Actually, I think that all of the attributes should be called when constructed (i.e., Damage, elevation, Occupied, and Passable). 

Right now, any collectible that is called will have an occupied value of false and an elevation of 0.

Reply:
I am changing my mind now after I reviewed the other base classes. I think the default values are good. You can ignore this comment lol.